vdist/configuration.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import
+
+import string
+import sys
+# The ConfigParser module has been renamed to configparser in Python 3
+if sys.version_info[0] == 3:
+    import configparser
+else:
+    import ConfigParser as configparser
+
+import vdist.source as source
+
+LISTABLE_ARGUMENTS = {"source_git", "source_git_directory", "runtime_deps",
+                      "build_deps"}
+LONG_TEXT_ARGUMENTS = {"fpm_args", "pip_args"}
+PROCESSABLE_ARGUMENTS = {"output_folder", "source_directory", "compile_python",
```

Comment:
`LISTABLE_ARGUMENTS` are those that has to be converted to a list in order to be processed.

`LONG_TEXT_ARGUMENTS` are arguments that can include carriage returns. This character can create some troubles so I remove them before storing the argument in `Configuration` instance.

`PROCESSABLE_ARGUMENTS` is a superset that includes LISTABLE and LONG_TEXT sets besides other parameters that are not in the prior ones. The point here is that PROCESSABLE _ARGUMENTS cannot be directly imported into Configuration.builder_parameters without being processed before.